src/Servers/Kestrel/Core/src/Features/IHttpMinRequestBodyDataRateFeature.cs

```diff
@@ -17,6 +17,6 @@ public interface IHttpMinRequestBodyDataRateFeature
         /// This feature is not supported for HTTP/2 requests except to disable it entirely by setting <see cref="MinDataRate"/> to <see langword="null"/> 
         /// Instead, use <see cref="KestrelServerLimits.MinRequestBodyDataRate"/> for server-wide configuration which applies to both HTTP/2 and HTTP/1.x.
         /// </summary>
-        MinDataRate MinDataRate { get; set; }
+        MinDataRate? MinDataRate { get; set; }
```

Comment:
Can we make this non-nullable.

Reply:
It is set to null in HTTP/2

Reply:
I'd prefer if the feature didn't work at all in HTTP/2

Reply:
You think there should never be a min request data rate for HTTP/2? Or you think there shouldn't be a way to disable it for HTTP/2?

You should create a new issue. Right now this property definitely supports being set to null. gRPC does it for streaming.

Reply:
I think the feature should either exist and work or not exist. Right now the feature is on and we don't support it for HTTP/2. 

This feature solely exists for that property so its strange that the feature is returned then the property is null

Reply:
It does exist.

https://github.com/dotnet/aspnetcore/pull/28050#discussion_r530711619